### PR TITLE
remove oats_can_interfaces variable

### PR DESCRIPTION
**Pull request by aaron97neu**
Should fix sleep cycling issue

### CODE REVIEW

File: ansible/avena/roles/can/templates/system-sleep/reset-can-interfaces.j2 [hunk 1]
```diff
@@ -4,7 +4,7 @@ echo "Reset can interfaces"
 case $1/$2 in
         post/*)
                 echo "Waking up from $2..."
-                {% for interface in oats_can_interfaces %}
+                {% for interface in can_interfaces %}
                 systemctl restart can@{{ interface }}
```
Comment (abalmos) [inline ansible/avena/roles/can/templates/system-sleep/reset-can-interfaces.j2:8]:
```suggestion
                systemctl restart can@{{ interface.name }}
```

**Review by abalmos — APPROVED** [review]
Looks right less that one loop

**Review by facastiblancor — APPROVED** [review]
Looks good. Let's wait for some testing before merging.

**Comment by aaron97neu** [comment]
Tested on two boards in the lab, should be good to merge

**Pull request merged**